spec/helpers/application_helper_spec.rb

```diff
@@ -1,7 +1,10 @@
 require 'rails_helper'
 
 describe ApplicationHelper, type: :helper do
-   
+  before do
```

Comment:
Should this go in an RSpec context instead? This would allow us to test the behavior with or without the gem.